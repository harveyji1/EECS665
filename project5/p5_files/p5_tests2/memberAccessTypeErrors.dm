Person : class{
	a : int;
	b : bool;
	fn : () bool {}
	func : (c : int, d : bool) int {}
};

main : () void 
{
	z : int; 
	t : Person;
	
	z = t--func(true, 1);
	z = t--func(1, true);
	z = t--a;
	z = t--b;
	z = t--fn();
}