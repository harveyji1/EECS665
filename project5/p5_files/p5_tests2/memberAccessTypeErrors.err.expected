FATAL [13,14]-[13,18]: Type of actual does not match type of formal
FATAL [13,20]-[13,21]: Type of actual does not match type of formal
FATAL [16,2]-[16,10]: Invalid assignment operation
FATAL [17,2]-[17,13]: Invalid assignment operation
Type Analysis Failed
